.storybook/main.ts

```diff
@@ -23,22 +42,30 @@ module.exports = {
         'sass-loader',
       ],
     });
-    config.performance = {
-      hints: false, // https://webpack.js.org/configuration/performance/#performancehints
-    };
+
+    // https://webpack.js.org/configuration/performance/#performancehints
+    config.performance = { hints: false };
+
+    // resolve @onfido/castor imports
```

Comment:
Maybe better:

```suggestion
    // resolve Castor imports
```

Because `@onfido/castor` seems more like a core package (without React one).